PapersCited.py

```diff
@@ -152,6 +153,29 @@ def _adjust_common_phrases(self):
                 clean_citations[index_no] = clean_citations[index_no].replace(key, phrases_to_adjust[key])
         return(clean_citations)
     
+    def _separate_multiple_years(self):
+        rx = RegexPatterns()
+        all_citations = self.citations
+        # For each citation, find how many years it contains.
+        # If it contains more than one, get the years as a list.
+        # Flag the original citation for deletion
+        single_year_pattern = "\\d{4}[abcd]?"
+        all_years_pattern = rx.years
+        extracted_citations = []
+        
+        for index_no, citation in enumerate(all_citations):
+            years = regex.findall(pattern = single_year_pattern, string = citation)
+            # findall always returns a list
+            if len(years) > 1:

```

Comment:
Does it make sense to test separately what happens in the case if len(years) > 1 is false